app/components/templates/templatesController.js

```diff
@@ -179,6 +179,8 @@ function ($scope, $q, $state, $filter, Config, Info, Container, ContainerHelper,
       var selectedTemplate = $scope.templates[id];
       $scope.state.selectedTemplate = selectedTemplate;
       $scope.formValues.ports = selectedTemplate.ports ? TemplateHelper.getPortBindings(selectedTemplate.ports) : [];
+      $location.hash('selectedTemplate');
+      $anchorScroll();
```

Comment:
Sure, I tested with only `$anchorScroll('selectedTemplate');` and it works well :-)